### DISCUSSION TITLE
How to set pagination_per_page for archive page?

### DISCUSSION THREAD

**Discussion by shenweiyan** [squidfunk/mkdocs-material#6376] [Q&A]
I've found that the `pagination_per_page` parameter in the blog plugin affects both the blog's main page and the archive pages. 

Now, I want to customize a different pagination limit specifically for the archive pages, for example, setting `pagination_per_page=20` for the archives page while keeping `pagination_per_page=5` for the main blog page. 

Is there a way to achieve this?

**Comment by squidfunk** [comment]
That's currently not supported, but you can open a change request if you like.
